tox.ini

```diff
@@ -0,0 +1,26 @@
+;
```

Comment:
can this file be somewhere outside the root directory?
perhaps sdks/python/ or sdks/python/tools, or something similar?